FreeSurfer/custom/fs_to_gii

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Per a comment that I saw Tim made for `test_resample_fs`, if this is just a helper script, and not making a "necessary change to freesurfer's behavior", it probably belongs in `scripts` rather than `custom`.

Reply:
More to the point, if it isn't actually called by the pipeline (and also not by recon-all or other indirect methods), and is just a convenience function for interactive use, then it probably shouldn't be committed to the repository.

Reply:
I removed it.